scripts/workers/code_upload_worker_utils/network_policies.yaml

```diff
@@ -8,6 +8,7 @@ spec:
       {}
   egress:
   - toFQDNs:
+    - matchName: archive.ubuntu.com
```

Comment:
Different values for the same parameter name `matchName`?

Reply:
@Ram81 ?

Reply:
@RishabhJain2018 we are adding multiple entries for `matchName` to allow access to different FQDNs. We need access to `archive.ubuntu.com` for our pre-start hook (the hook that updates submission evaluation started at)

Reply:
I see..okay.